yarn-project/noir-contracts/src/scripts/copy_output.ts

```diff
@@ -34,6 +34,8 @@ function writeToProject(abi: any) {
       const toWrite = {
         ...abi,
         functions: abi.functions.map((f: any) => omit(f, projectContract.exclude)),
+        // If we maintain debug symbols they will get commited to git.
```

Comment:
I tried untracking the account contracts but it generates a chicken and egg issue, where noir-contracts depends on aztec.js but it writes assets to aztec.js so if the account contracts abis are untracked noir-contracts will fail to build and thus to generate the account contract abis :cry:

Reply:
Can we move around the deps to avoid this? some packages can depend on the `noir-contracts` package and copy the built artifacts over

Reply:
I'm not sure we can move the deps around easily, because if aztec.js depends on compilation results then compilation cannot depend on aztec.js. I think the best solution would be for noir-contracts to not depend on aztec.js, but for that I think we'd need to remove types generation out of noir_contracts